vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
> Looks like this can all be reduced to:

it might, but this is not this particular patch purpose, if you/ @Fryguy prefer a style change, I can certainly do it in a separate patch.